.travis.yml

```diff
@@ -1,4 +1,7 @@
-sudo: false
+sudo: required
+
+services:
+  - docker
```

Comment:
I added Docker to travis, thinking I might switch it to use the dockerized build.  It's fine for now.